go-mfa-example/static/stylesheets/style.css

```diff
@@ -42,24 +36,16 @@ body {
   align-items: center;
 }
 
-.space_evenly {
-  justify-content: space-evenly;
-}
-
-.space_between {
+.space-between {
   justify-content: space-between;
 }
 
-.width_40vw {
-  width: 40vw;
-}
-
-.width_10vw {
-  width: 10vw;
+.width-75 {
+  width: 75%;
 }
 
-.mt-20 {
-  margin-top: 20px !important;
+.width-40vw {
+  width: 40vw;
 }
 
 .container_success {
```

Comment:
On line 81 in the button class can you change the padding to 8 and 16? We're getting rid of all the big buttons :)